js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala

```diff
@@ -107,6 +107,8 @@ abstract class ExternalJSEnv(
       for ((name, value) <- vmEnv)
         pBuilder.environment().put(name, value)
 
+      logger.debug("Starting process: " + allArgs.mkString(" "))
+
```

Comment:
Could you put this in a separate commit with a separate commit message? (It just might be a bit confusing after the fact that "increasing the Node.js heap" changes what the envs log).